pkg/k8s/identitybackend/identity.go

```diff
@@ -47,7 +50,7 @@ func NewCRDBackend(c CRDBackendConfiguration) (allocator.Backend, error) {
 
 type CRDBackendConfiguration struct {
 	NodeName string
-	Store    cache.Store
+	Store    cache.Indexer
 	Client   clientset.Interface
 	KeyType  allocator.AllocatorKey
```

Comment:
I realize this is probably too much of a refactor for this PR, but it seems like `KeyType` could be replaced with `func(map[string]) allocator.AllocatorKey` - it would be more idiomatic.

Only if you're in the mode for a bit more work :-)

Reply:
Changed it to KeyFun accepting func(map[string]string) allocator.AllocatorKey